pyccl/neutrinos.py

```diff
@@ -25,41 +32,49 @@ def Omeganuh2(a, Neff, mnu, TCMB=2.725):
     # Convert to array if it's not already an array
     if not isinstance(a, np.ndarray):
         a = np.array([a,]).flatten()
+    if not isinstance(mnu, np.ndarray):
+        mnu = np.array([mnu,]).flatten()
+    
+    # FIXME: Implement length checking of mnu (should it be a certain length?)
     
     # Call function
-    OmNuh2, status = lib.Omeganuh2_vec(Neff, mnu, TCMB, a, a.size, status)
+    OmNuh2, status = lib.Omeganuh2_vec(Neff, TCMB, a, mnu, a.size, status)
     
     # Check status and return
     check(status)
     if scalar: return OmNuh2[0]
     return OmNuh2
 
 
-def Omeganuh2_to_Mnu(a, Neff, OmNuh2, TCMB=2.725):
+def nu_masses(OmNuh2, label, ccl_TCMB=2.725):
     """Omeganu2h_to_Mnu
 
     Args:
-        a (float or array_like): Scale factor(s), normalized to 1 today.
-        Neff (float): Number of relativistic neutrino species (NB: to all practical purposes, Neff is simply N_nu_mass)
-        OmNuh2 (float): Neutrino energy density times h^2
+        OmNuh2 (float): Neutrino energy density at z=0 times h^2
+        label: indicates how the masses should be split up
```

Comment:
Just flagging up this use of the word `label`, in case the terminology changes (see previous comments).

Reply:
I changed this to "mass_split" to be more description.